keyboards/zlant/readme.md

```diff
@@ -0,0 +1,15 @@
+# Zlant
+
+![Zlant](https://i.imgur.com/Siz8qsL.jpg)
+
+A compact 40% inspired by the Plank with a staggered layout instead.
+
+Keyboard Maintainer: [Felipe Coury](https://github.com/fcoury)
```

Comment:
You want an extra line between these.  Not sure why the formatting messes up like this.

Reply:
Fixed.